.travis.yml

```diff
@@ -1,8 +1,8 @@
 jobs:
   include:
-   - os: osx
+   - os: linux
 language: node_js
-node_js: '10'
+node_js: '12'
```

Comment:
Not sure. Why do we need this?

Reply:
I just change to LTS recommended here https://nodejs.org/en/ . But we can still use 10.X version